src/collection/views/EditCollectionMetadata.tsx

```diff
@@ -27,79 +34,153 @@ const EditCollectionMetadata: FunctionComponent<EditCollectionMetadataProps> = (
+											<TagsInput
+												options={(data.vocabularies_lom_contexts || []).map(item => ({
+													value: item.label,
+													label: item.label,
+												}))}
+												value={(collection.lom_context || []).map((item: string) => ({
+													value: item,
+													label: item,
+												}))}
+												onChange={(values: TagInfo[]) =>
+													updateCollectionMultiProperty(values, 'lom_context')
+												}
+											/>
+										</FormGroup>
+										<FormGroup label="Vakken" labelFor="subjectsId">
+											{/* TODO get subjects from the database once the table is filled in */}
+											<TagsInput
+												options={[]}
+												// onChange={(values: TagInfo[]) =>
+												// 	updateCollectionMultiProperty(values, 'subjects')
+												// }
+											/>
+										</FormGroup>
+										<FormGroup
+											label="Korte omschrijving"
+											labelFor="shortDescriptionId"
+											error={getValidationFeedbackForShortDescription(collection, true)}
+										>
+											<TextArea
+												name="shortDescriptionId"
+												value={collection.description || ''}
+												id="shortDescriptionId"
+												height="medium"
+												onChange={(value: string) => updateCollectionProperty(value, 'description')}
+											/>
+											<label>{getValidationFeedbackForShortDescription(collection)}</label>
+										</FormGroup>
+										<FormGroup
+											label="Persoonlijke opmerkingen/notities"
+											labelFor="personalRemarkId"
+										>
+											<TextArea
+												name="personalRemarkId"
+												value={''}
+												id="personalRemarkId"
+												height="medium"
+												placeholder="Geef hier je persoonlijke opmerkingen/notities in..."
+											/>
+										</FormGroup>
+									</Column>
+									<Column size="3-5">
+										<FormGroup label="Cover afbeelding" labelFor="coverImageId">
+											<Button
+												type="secondary"
+												label="Stel een afbeelding in..."
+												onClick={() => setShowCoverImageModal(true)}
+											/>
+										</FormGroup>
+										<FormGroup label="Map" labelFor="mapId">
+											<Button type="secondary" icon="add" label="Voeg toe aan een map" />
+										</FormGroup>
+									</Column>
+								</Grid>
+							</Spacer>
+						</Form>
+					</Container>
 				</Container>
-			</Container>
+				<Modal
+					isOpen={showCoverImageModal}
+					title="Stel een cover afbeelding in"
+					size="large"
+					onClose={() => setShowCoverImageModal(false)}
+					scrollable={true}
+				>
+					<ModalBody>
+						<div className="u-spacer">
+							<Form>
+								<ImageGrid
+									images={getCollectionStills()}
+									allowSelect={true}
+									value={selectedCoverImages}
+									onChange={setSelectedCoverImages}
+									width={100}
+									height={100}
+								/>
+							</Form>
+						</div>
+					</ModalBody>
+					<ModalFooterLeft>
```

Comment:
In de meeste modals staan de actions naast elkaar rechts, mss best hier ook doen?

Reply:
fixed + ook gefixed voor 2 andere modals met hetzelfde probleem